telegram/message.py

```diff
@@ -281,6 +293,15 @@ class Message(TelegramObject):
         proximity_alert_triggered (:class:`telegram.ProximityAlertTriggered`): Optional. Service
             message. A user in the chat triggered another user's proximity alert while sharing
             Live Location.
+        voice_chat_started (:class:`telegram.VoiceChatStarted`, Optional): Represents a service
```

Comment:
```suggestion
        voice_chat_started (:class:`telegram.VoiceChatStarted`): Optional. Represents a service
```
same below. The optional needs to be in the description for attributes, b/c otherwise it renders badly

Reply:
Hmm right, thanks